### PR TITLE
Fix evdev threading

### PR DESCRIPTION
**Pull request by Tilka**
stuff, idk, I'm tired

### CODE REVIEW

File: Source/Core/DolphinWX/ISOProperties/ISOProperties.cpp [hunk 1]
```diff
@@ -349,7 +349,6 @@ long CISOProperties::GetElementStyle(const char* section, const char* key)
 void CISOProperties::CreateGUIControls()
 {
   const int space5 = FromDIP(5);
-  const int space10 = FromDIP(10);
```
Comment (ligfx) [inline Source/Core/DolphinWX/ISOProperties/ISOProperties.cpp]: This is unrelated

File: Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp [hunk 2]
```diff
@@ -116,32 +116,33 @@ static void HotplugThreadFunc()
...
-    }
-    s_hotplug_thread.join();
   }
+  s_hotplug_thread.join();
 }
 
 void Init()
 {
   s_devnode_name_map.clear();
-  PopulateDevices();
```
Comment (Tilka) [inline Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp]: @ligfx: https://github.com/dolphin-emu/dolphin/commit/3e69d066f5e9ad0a1097d806c656035398bbb7bc#diff-766fe5f4c3684c597c10bd18606179caR117

Comment (ligfx) [inline Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp]: Oh, I forgot that gets called even for hotplug backends. Does it cause a hang when `RefreshDevices` is called then..?

Comment (Tilka) [inline Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp]: iirc, yes

**Comment by degasus** [comment]
leoetlino | it looks good to me, but I can't test it since I'm not affected by the issue
